be/src/exec/pipeline/crossjoin/cross_join_context.h

```diff
@@ -35,7 +35,7 @@ class CrossJoinContext final : public ContextWithDependency {
               _rf_hub(params.rf_hub),
               _rf_descs(std::move(params.rf_descs)) {}
 
-    void close(RuntimeState* state) override {}
+    void close(RuntimeState* state) override { _build_chunks.clear(); }
```

Comment:
Better to implement this function in source files, in order to reduce the header dependence of this file.

Reply:
done